.devcontainer/scripts/configure-proxies.sh

```diff
@@ -17,11 +17,17 @@ echo "#######################################################"
 echo "### Configure-proxies                               ###"
 echo "#######################################################"
 
-# this is needed for K3D to work properly
-NO_PROXY="localhost,127.0.0.1,0.0.0.0,10.0.0.0/8,192.168.122.0/24,cattle-system.svc,.svc,.cluster.local"
+if [ "${PROXY}" = ".Proxy" ]; then
+    USE_PROXIES="true"
+    HTTP_PROXY="${PROXY_HOST}:${PROXY_PORT}"
+    HTTPS_PROXY=$HTTP_PROXY
+    FTP_PROXY=$HTTP_PROXY
+    ALL_PROXY=$HTTP_PROXY
+    NO_PROXY="localhost,127.0.0.1,0.0.0.0,10.0.0.0/8,192.168.122.0/24,cattle-system.svc,.svc,.cluster.local"
+    CONFIGURE_GIT="true"
+fi
```

Comment:
Is that use during container build or for running it? If it's for building isn't it duplication the above stuff (from dockerfile)?